dmgroups.tex

```diff
@@ -142,21 +142,21 @@ \subsection{DM Change Control Board \label{sect:dmccb}}
 
 \begin{itemize}
 \item Responsibilities:
-	\begin{itemize}
-	\item Determine when deliverables (controlled documents and software) are ready to be baselined (placed under configuration controlled status) or released. This include LDM series documents.
-	\item Review and approve/reject proposed changes to baselined items
-        \item Review and approves all RFCs prior to 'Adoption'.
-	\item Monitor and approve  DM software releases
-        \item Monitor DM Jira issues status.
-	\item Ensure that the DM  follows  LSST and DM configuration control processes.
-	\end{itemize}
+        \begin{itemize}
+        \item Determine when deliverables (controlled documents and software) are ready to be baselined (placed under configuration controlled status) or released. This include LDM series documents.
+        \item Review and approve/reject proposed changes to baselined items
+        \item Review and approve all RFCs prior to 'Adoption'.
+        \item Monitor and approve DM software releases
+        \item Monitor the status of issues in the DM project on Jira.
+        \item Ensure that the DM follows LSST and DM configuration control processes.
+        \end{itemize}
 \item Membership:
-	\begin{itemize}
+        \begin{itemize}
         \item Core members:
                 \begin{itemize}
                 \item DM Project Manager
                 \item DM Subsystem Scientist
-	        \item Chaired by the Systems Engineer (\secref{role:sysengineer}).
+                \item Systems Engineer, Chair (\secref{role:sysengineer}).
```

Comment:
Can we write “Systems Engineer (chair)”, to be consistent with “DM Configuration and Release Manager (secretary)”? (Or alternatively, “DM Configuration and Release Manager, Secretary)”, if you prefer.

Reply:
OK for the second option.